### DISCUSSION TITLE
Is there something like this but for Windows?

### DISCUSSION THREAD

**Discussion by strawbberrys** [Schneegans/Fly-Pie#219] [Fly-Pie Q&A] [answered]
I use this on my Linux machine and I've gotten so used to it that it feels weird not having it on my Windows machine. So, is there anything like this extension but for Windows? Windows 10 specifically.

**Comment by Schneegans** [comment]
At least, there's nothing I am aware of. If anybody finds something, I would be happy to learn about it as well!

**Comment by Schneegans** [comment, marked as answer]
I just published some [news regarding a platform-agnostic pie menu](https://ko-fi.com/post/Introducing-Ken-Do-L3L7L0FQ2) I am working on! This will also work on Windows! At the end of the post, I wrote down some ways how you could contribute to the project. I would be more than happy if you would follow the progress and support the development along the way :smile:
